src/main/java/edu/bu/metcs673/trackr/api/TransactionDTO.java

```diff
@@ -0,0 +1,33 @@
+package edu.bu.metcs673.trackr.api;
+
+import edu.bu.metcs673.trackr.common.CommonConstants;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Validated
+public class TransactionDTO {
+
+	@NotNull(message = CommonConstants.TRANSACTION_MONEY)
+	private long bankAccountId;
+
+	@NotNull(message = CommonConstants.TRANSACTION_MONEY)
+	private double money;
```

Comment:
Add the `@PositiveOrZero` annotation so that we ensure that we only accept positive values.

Reply:
I think the transaction can be negative number.
For example:
You give some to me. this time the money is positive
I give some to you. this time the money is negative

I don't know whether this is right?

Reply:
Ok that make sense.  I think i was considering having some kind of "TRANSACTION_TYPE" column and then add/subtracting based on that value.  I think your way is easier.